### DISCUSSION TITLE
Are Project Admins allowed to contribute on their own project?

### DISCUSSION THREAD

**Discussion by rajnishdesai23** [sanjay-kv/Open-source-Practice#995] [Q&A] [answered]
i saw some of the PAs in leaderboard and they have contribution of their own repo. so is it allowed or not
beacuse i saw their contribution they have not raised any issue just made PR and self assigned it and add labels himself

**Comment by sanjay-kv** [comment, marked as answer]
No they can contribute , but it wont count towards leaderboard.

**Comment by codeftsaikat** [comment]
Yes, Project Admins are allowed to contribute to their own project.
